referenceframe/transformable.go

```diff
@@ -31,6 +31,14 @@ func NewPoseInFrame(frame string, pose spatialmath.Pose) *PoseInFrame {
 	}
 }
 
+// NewZeroPoseInFrame is a convenience method that creates a PoseInFrame with the specified Frame and a zero pose.
+func NewZeroPoseInFrame(frame string) *PoseInFrame {
```

Comment:
Personally I find this a lot easier. If I'm writing something somewhere outside an IDE I no longer have to remember argument order, and it's a lot shorter.

Reply:
Its a few characters shorter. Normally I'm in favor of helper functions where they are useful but this doe not decrease line count and proliferates the number of public functions in the package.

Reply:
> this does not decrease line count

Actually it does, because creating zero poses in frame is a major cause of linter-forced line length splits. :)

Regardless, the major value here is reduced developer mental load in the *very* common case of creating a zero pose in frame.